subworkflows/local/samplesheet_to_channel/main.nf

```diff
@@ -309,7 +309,7 @@ workflow SAMPLESHEET_TO_CHANNEL {
     if (step == 'mapping' && umi_read_structure && umi_in_read_header) {
         // If UMIs are in read header, then we cannot use umi_read_structure separately, and instead the UMIs will be taken directly from the header
         // This requires us to set umi_read_structure to "+T +T " to indicate that UMIs are in the read header
-        if( umi_in_read_header != "+T +T ") {
+        if( umi_read_structure != "+T +T" ) {
```

Comment:
are you sure you want to remove the space at the end? if so, please also update the comment

Reply:
[fbc806e](https://github.com/nf-core/sarek/pull/2075/commits/fbc806e58db9a42e8e4785fac9b815a6d1c57ae0)